pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.black]
+line-length = 88
+target-version = ["py311"]
+
+[tool.isort]
+profile = "black"
+line_length = 88
+known_first_party = ["odh_data_processing"]
+src_paths = ["kubeflow-pipelines"]
```

Comment:
`notebooks` should be added here since there are python utility files in the repo.